src/vast_tracker.js

```diff
@@ -65,7 +65,7 @@ export class VASTTracker extends EventEmitter {
     // Nonlinear and companion creatives provide some tracking information at a variation level
     // While linear creatives provided that at a creative level. That's why we need to
     // differentiate how we retrieve some tracking information.
-    if (this.creative instanceof CreativeLinear) {
+    if (this.creative instanceof createCreativeLinear) {
```

Comment:
This will not work, probably need a function next to `createCreativeLinear` which is `isCreativeLinear` and this will check that this plain object contain all linear creative property